chrome/content/zotero/xpcom/prefs.js

```diff
@@ -252,6 +252,15 @@ Zotero.Prefs = new function(){
 			else {
 				Zotero.Sync.EventListeners.AutoSyncListener.unregister();
 				Zotero.Sync.EventListeners.IdleListener.unregister();
+				Zotero.Prefs.set('sync.reminder.autoSync.enabled', true);
+				// We don't want to immediately display reminder so bump this value
+				Zotero.Prefs.set('sync.reminder.autoSync.lastDisplayed', String(Date.now()));
```

Comment:
Why seconds instead of milliseconds? Milliseconds definitely store unnecessary information, but they make comparisons so much simpler, and we aren't at risk of overflow here.

Reply:
I dunno…it just feels a little messy to store the extra numbers. All of our timestamps are seconds, and most (though not all) of the timestamps Mozilla stores are too.

You're saying you're not concerned with the literally dozens of collective megabytes that will be taken up on storage devices around the world?